README.rst

```diff
@@ -29,57 +31,89 @@ Further documentation is available on
 https://invenio-curations.readthedocs.io/
 
 
+Update ``invenio.cfg``
+----------------------
 
-## Update invenio.cfg
-### Add notification builders and entity resolver for groups
+Add `notification builders` and `entity resolver` for `groups`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-from invenio_curations.notifications.builders import (
-    CurationRequestAcceptNotificationBuilder,
-    CurationRequestSubmitNotificationBuilder,
-)
+Currently, requests can only be sent to a single `receiver`.
+However, curation reviews are typically performed by a `group` of people rather than one single fixed `user`.
+Thus, the curation requests are sent to a `group` rather than a single `user` in the system so that all users with a certain `role` can receive and act on curation requests.
 
-from invenio_records_resources.references.entity_resolvers import ServiceResultResolver
+To enable resolution of user groups, an `entity resolver` has to be added to the configuration.
 
-from invenio_app_rdm.config import NOTIFICATIONS_BUILDERS, NOTIFICATIONS_ENTITY_RESOLVERS
-NOTIFICATIONS_BUILDERS = {
-    **NOTIFICATIONS_BUILDERS,
-    # Curation request
-    CurationRequestAcceptNotificationBuilder.type: CurationRequestAcceptNotificationBuilder,
-    CurationRequestSubmitNotificationBuilder.type: CurationRequestSubmitNotificationBuilder,
-}
+Additionally, notification builders have to be configured so that notifications are sent out to the involved users whenever something's happening in the curation review.
 
-NOTIFICATIONS_ENTITY_RESOLVERS = NOTIFICATIONS_ENTITY_RESOLVERS + [ServiceResultResolver(service_id="groups", type_key="group")]
+.. code-block:: python
```

Comment:
note: the empty line between this and the (indented) code block is required; omitting it won't render any code block.